### ISSUE TITLE
Setup fail (with analytics)

### ISSUE THREAD

**Issue by doublejosh** [prescottprue/generator-react-firebase#164]
The app cannot be compiled or run *after* setup if choosing to include analytics. Two issues thus far...

```
./src/utils/analytics.js
Attempted import error: 'segmentId' is not exported from 'config' (imported as 'config').
```

```
./src/utils/analytics.js
  Line 50:28:  'eventName' is not defined                         no-undef
  Line 52:7:   'GOOGLE_EVENT_TYPES_BY_EVENT_NAME' is not defined  no-undef
  Line 52:40:  'eventName' is not defined                         no-undef
  Line 52:54:  'eventName' is not defined                         no-undef
  Line 55:47:  'eventName' is not defined                         no-undef
```

**Possible Solutions**
Add a segment ID input to the init process.

The analytics constants file doesn't exist, it should be created here...
`/src/constants/analytics.js`

This line may be incorrect...
`src/utils/analytics.js`
```javascript
export function triggerAnalyticsEvent(eventNameKey, eventData) {
```
...I believe `eventNameKey` should be `eventName`

This variable `GOOGLE_EVENT_TYPES_BY_EVENT_NAME` may need to be the same as the import `ANALYTICS_EVENT_NAMES`

**Comment by prescottprue** [comment]
Oh man that is a silly one, thanks for reporting!

**Comment by prescottprue** [comment]
Fixed in [v4.0.1 release](https://github.com/prescottprue/generator-react-firebase/releases/tag/v4.0.1), thanks for reporting!
